lib/pbench/cli/agent/commands/results/move.py

```diff
@@ -175,7 +166,7 @@ def execute(self, single_threaded: bool, delete: bool = True) -> int:
         return 0 if failures == 0 else 1
 
 
-@click.command()
+@click.command(name="pbench-results-move")
```

Comment:
I don't think this accomplishes anything useful.

That is, for external purposes, I think the command name is set in `setup.cfg`.  For local purposes, I expect that we're going to want the command name to be just `"move"`, because it will be a sub-command of the `"results"` subcommand.

Reply:
Exception raised when no `name` attribute.

Reply:
See [below](https://github.com/distributed-system-analysis/pbench/pull/2325#discussion_r658172074).

Reply:
I removed the explicit `name` attribute, and the `name` becomes `main`, the name of the function.

For now, until we implement the full use of click commands with a hierarchy, etc., we want the "usage" string to say: "Usage: pbench-results-move ..." and "Usage pbench-results-push ...".